README.md

```diff
@@ -64,7 +64,7 @@ _Note: this library is published to work on Java 8 and above. However, you will
 ### CLI Usage
 
 The `smithytranslate` CLI will recursively go through all child directories of the
-input directory provided and format any Smithy files it finds. The output 
+input directory provided and format any Smithy files it finds. The output
```

Comment:
unrelated to your PR, but it seems like `the output` is just a leftover incomplete sentence